index.html

```diff
@@ -353,6 +354,14 @@
             </li>
           </ul>
         </section>
+        <section id="httpGETLDPRS">
+          <h2><a>LDP-RS</a>s</h2>
+          <p>
+            <code>GET</code> requests that include a <code>Prefer</code> header to any <a>LDP-RS</a> MUST
```

Comment:
Application of the preference is optional (MAY and SHOULD above), so shouldn't this be:

> Responses to <code>GET</code> requests that apply a <code>Prefer</code> request header to any <a>LDP-RS</a> MUST include the <code>Preference-Applied</code> response header as defined in RFC7240 section 3.